clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLWriter.java

```diff
@@ -141,6 +142,64 @@ public void writeContent(XMLExtendedStreamWriter writer, SubsystemMarshallingCon
                     writer.writeEndElement();
                 }
             }
+
+            if (model.hasDefined(RemoteCacheContainerResourceDefinition.WILDCARD_PATH.getKey())) {
+                for (Property entry : model.get(RemoteCacheContainerResourceDefinition.WILDCARD_PATH.getKey()).asPropertyList()) {
+
+                    String remoteContainerName = entry.getName();
+                    ModelNode remoteContainer = entry.getValue();
+
+                    writer.writeStartElement(XMLElement.REMOTE_CACHE_CONTAINER.getLocalName());
+                    writer.writeAttribute(XMLAttribute.NAME.getLocalName(), remoteContainerName);
+
+                    writeAttributes(writer, remoteContainer, EnumSet.allOf(RemoteCacheContainerResourceDefinition.Attribute.class));
+
+                    if (remoteContainer.hasDefined(ClientThreadPoolResourceDefinition.WILDCARD_PATH.getKey())) {
```

Comment:
This is a required child, so this resource is guaranteed to exist.

Reply:
Fixed, removed.